integration/monai/examples/spleen_ct_segmentation_local/jobs/spleen_ct_segmentation_local/app/config/config_fed_server.json

```diff
@@ -2,7 +2,7 @@
   "format_version": 2,
 
   "min_clients": 2,
-  "num_rounds": 2,
+  "num_rounds": 100,
```

Comment:
2 should be fine just to showcase it works?

Reply:
the instructions in the README has a step to change the num_rounds to a smaller value to make it faster, so I feel like it should be higher like a "real" example to begin with